rules/gsuite_activityevent_rules/google_workspace_advanced_protection_program.yml

```diff
@@ -0,0 +1,76 @@
+AnalysisType: rule
+Description: 'Your organization''s Google Workspace Advanced Protection Program settings were modified. '
```

Comment:
This does not need the wrapping single quotes. `Description: Your org's workspace thing ok.` should be sufficient